src/librustc/middle/resolve_lifetime.rs

```diff
@@ -150,7 +149,7 @@ impl Region {
         }
     }
 
-    fn subst(self, params: &[hir::Lifetime], map: &NamedRegionMap) -> Option<Region> {
+    fn subst(self, params: Vec<&hir::Lifetime>, map: &NamedRegionMap) -> Option<Region> {
```

Comment:
This seems potentially expensive, why not use an iterator?

Reply:
Ah, I thought I had replaced these all with iterators; must have overlooked that one.